examples/20_calculate_farm_power_with_uncertainty.py

```diff
@@ -16,41 +15,106 @@
 """
 
 # Instantiate FLORIS using either the GCH or CC model
-fi = FlorisInterface("inputs/gch.yaml") # GCH model
-fi_unc = UncertaintyInterface("inputs/gch.yaml") # Add uncertainty with default settings
+fi = FlorisInterface("inputs/gch.yaml")  # GCH model
+fi_unc_3 = UncertaintyInterface(
+    "inputs/gch.yaml", verbose=True, wd_std=3
+)  # Add uncertainty with default settings
```

Comment:
Is this default settings with the different `wd_std`?

Reply:
3 is the default, but passing in to be explicit

Reply:
Ah I see what you mean, fixing